### PR TITLE
Added missing kernel IO headers.

### PR DESCRIPTION
**Pull request by Rinnegatamante**
Added missing kernel IO headers.

### CODE REVIEW

**Comment by xerpi** [comment]
Looks good to me.

**Comment by d3m3vilurr** [comment]
looks good to me either.
but just curious, anyone can explain why we broke CamelCase?
eg;
- `ksceIoDevctl` vs `ksceIoDevCtl`
- `ksceIoGetstatByFd` vs `ksceIoGetStatByFd`

**Comment by xerpi** [comment]
The userspace functions are
```
sceIoDevctl
```
and
``` 
ksceIoGetstatByFd
```
so not sure if they are wrong.

**Comment by TheOfficialFloW** [comment]
Sony defined them this way

File: include/psp2kern/io/devctl.h [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/**
+ * \kernelgroup{SceDevCtl}
+ * \usage{psp2kern/io/devctl.h,?}
```
Comment (devnoname120) [inline include/psp2kern/io/devctl.h]: Please change `?` to `-lstub_name`.
The `?` left in the include files are there out of laziness, it's not an example to follow.

**Comment by Rinnegatamante** [comment]
Should be fine now.

**Pull request merged**